pages/management/ManagementHandler.inc.php

```diff
@@ -169,6 +169,7 @@ function website($args, $request) {
 		$listsForm = new \PKP\components\forms\context\PKPListsForm($contextApiUrl, $locales, $context);
 		$privacyForm = new \PKP\components\forms\context\PKPPrivacyForm($contextApiUrl, $locales, $context, $publicFileApiUrl);
 		$themeForm = new \PKP\components\forms\context\PKPThemeForm($themeApiUrl, $locales, $context);
+		$dateTimeForm = new \PKP\components\forms\context\PKPDateTimeForm($contextApiUrl, $locales, $context);
```

Comment:
The locales here are the supported form locales, but I think that we need to use just `supportedLocales`. This will cover cases like when a date is displayed in the reader frontend.

Reply:
It's not allowed in `ValidatorFactory::allowedLocales`

Reply:
Can you explain what you mean in more detail? `ValidatorFactory::allowedLocales` should only work based on the locales that are passed into it, which could be any array of locales.

Reply:
When there is a language that isn't used in forms, the response upon saving the form is: 
```
{"dateFormatLong":{"xx_XX":["This language is not accepted."]}...
``

Reply:
Ahh, I see. The context handler validates the fields against `getSupportedFormLocales()` at https://github.com/pkp/pkp-lib/blob/master/api/v1/contexts/PKPContextHandler.inc.php#L348. Hmm, I'm trying to think if there is any case where this will cause a problem.

I think, if a journal has activated a `supportedLocales` (UI) that isn't active in `supportedFormLocales` (Form), a user can switch to that locale in the editorial backend and a request will be made for a date format in that locale... I guess in such circumstances it would fallback to what's defined in the config file, right? Maybe that's enough...

Would you be able to test that case to make sure it doesn't explode? I think it's ok if there's no customizable date format since very few journals will run this way, and if they want they can set a fallback in the config file.

Reply:
Yes, despite it's not possible to change the settings for the locale that isn't available for the form, it doesn't cause any issues when this locale is chosen in the UI - default settings take place as expected.